### DISCUSSION TITLE
MagicBell & Data Residency

### DISCUSSION THREAD

**Discussion by pnfb** [magicbell/community#121] [Questions] [answered]
Hello,

We use MagicBell as part of our SAAS product and we currently have a user that is interested in understanding where data (any type of it) is stored due to compliance/security policies. Could you please share what data residency looks like from the MagicBell perspective? Where are you storing data today?

Thank you in advance!

**Comment by pnfb** [comment, marked as answer]
Based on direct exchanges with MagicBell and in case someone else is interested:

MagicBell operates currently in the EU (Ireland) and has some subprocessors in the US, they can offer DPA but cannot offer customizations to the setup as of today.

**Reply by unamashana** [reply]
Thank you for posting that @pnfb.
